pkg/controller/daemon/daemon_controller.go

```diff
@@ -262,8 +285,9 @@ func (dsc *DaemonSetsController) Run(ctx context.Context, workers int) {
 
 	defer dsc.queue.ShutDown()
 
-	klog.Infof("Starting daemon sets controller")
-	defer klog.Infof("Shutting down daemon sets controller")
+	logger := klog.LoggerWithName(klog.FromContext(ctx), "DaemonSet")
```

Comment:
let's get a consensus between this PR and #113916.
Should we name the controller once in cmd/kcm/app/core.go?

Reply:
Yes.